### PR TITLE
Migrate from Travis to Github Actions

### PR DESCRIPTION
**Pull request by madwort**
* using pins from last known-good build https://travis-ci.com/github/ebmdatalab/datalabsupport/builds/96778036

### CODE REVIEW

File: datalabsupport/mail/tests.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def test_message_link_formatting(self, mock_slack):
 
         get_messages(folder='INBOX', channel='#mailtest')
         expected = ("<http://info.sagepub.co.uk/c/"
-                    "11o8zNX6zKkZTZfhD08iisMdxM3A|submission "
+                    "11o8zNX6zKkZTZfhD08iisMdxM3A|_blank|submission "
```
Comment (madwort) [inline datalabsupport/mail/tests.py:59]: @sebbacon this change makes the test pass, not sure if I need to dig in any further?

Comment (madwort) [reply datalabsupport/mail/tests.py:59]: the HTML goes into `lxml` via `BeautifulSoup`, seems like it's just a minor formatting issue & can ignore?

Comment (sebbacon) [reply datalabsupport/mail/tests.py:59]: I remember this being very hairy, but I haven't time to think about it again! :shipit:  and let's worry another time!

**Review by sebbacon — APPROVED** [review]

File: datalabsupport/mail/tests.py (hunk 1 above)
Comment (madwort) [reply datalabsupport/mail/tests.py:59]: 👍

**Pull request merged**